tests/unit/resources/standalone-help.txt

```diff
@@ -11,6 +11,9 @@ Options:
                         password for use with authentication and/or sudo
 
 
+The commands listed below are for the current mode.
+The current mode can be changed using the 'mode select' command.
```

Comment:
We're in agreement that the current UX leaves something to be desired, but I'll deal with it next week.